guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -197,6 +275,15 @@ If `eth0` was renamed to `em0`, call the installer to use the new interface name
 # leapp upgrade
 ----
 
+ifdef::satellite[]
```

Comment:
same as with `preupgrade`, I wouldn't make it an own step, but add it to the "real" `upgrade` step